rbd-target-api.py

```diff
@@ -2352,6 +2353,23 @@ def hostgroup(target_iqn, group_name):
                                                          api.response.json()['message'])), 400
 
 
+def fill_settings_dict(def_settings):
+    defaults = {}
+    limits = {}
+
+    for k, setting in def_settings.items():
+        # Return normalized value to match get_config()'s format
+        defaults[k] = getattr(settings.config, k)
+
+        if isinstance(setting, IntSetting):
+            limits[k] = {'min': setting.min_val, 'max': setting.max_val,
+                         'type': setting.type_str}
+        else:
```

Comment:
We should return the valid values for the `enum` type, something equivalent to:

```
elif isinstance(setting, EnumSetting):
    limits[k] = {'values': setting.valid_vals,
                 'type': setting.type_str}
```

Reply:
Fixed.